manifests/credentials.pp

```diff
@@ -22,12 +22,16 @@
   $private_key_or_path = '',
   $ensure = 'present',
   $uuid = '',
+  $unless_tries = 20,
+  $unless_try_sleep = 5,
```

Comment:
Instead of adding new params, how about reusing `$::jenkins::cli_tries` & `$::jenkins::cli_try_sleep` as is done in `jenkins::cli::exec`?